src/Nethermind/Nethermind.Serialization.Rlp/RlpBehaviors.cs

```diff
@@ -31,5 +31,6 @@ public enum RlpBehaviors
     /// </summary>
     InMempoolForm = 64,
     ExcludeHashes = 128,
-    All = AllowExtraBytes | ForSealing | Storage | Eip658Receipts | AllowUnsigned | SkipTypedWrapping | InMempoolForm | ExcludeHashes,
+    Eip7642Messages = 256,
```

Comment:
Will do without adding new `RlpBehaviors`, but this will need new decoder for `TxReceipt`, since bloom (de-)serialization happens there, not in `ReceiptsMessageSerializer`.

  Just a note, as I don't see any other protocol implementations adding new `IRlpDecoder`s.

Reply:
Done in https://github.com/NethermindEth/nethermind/pull/7052/commits/6f3240186249520f2677411733d00664bd21928e